src/hotspot/share/runtime/arguments.cpp

```diff
@@ -2218,18 +2218,79 @@ int Arguments::process_patch_mod_option(const char* patch_mod_tail, bool* patch_
       memcpy(module_name, patch_mod_tail, module_len);
       *(module_name + module_len) = '\0';
       // The path piece begins one past the module_equal sign
-      add_patch_mod_prefix(module_name, module_equal + 1, patch_mod_javabase);
+      add_patch_mod_prefix(module_name, module_equal + 1, false);
```

Comment:
`patch_mod_javabase` seems unused here now. Not obvious that this flag is being tracked properly with the new changes.

Reply:
Patch_mod_javabase is obsolete and is to be removed.
Each --patch-module is added to an array as the arguments are parsed.
After the arguments are parsed the array is checked to see if any modules have been patched.
That is sufficient for the decision to disable CDS.